### DISCUSSION TITLE
clickhouse jdbc error

### DISCUSSION THREAD

**Discussion by easy-data** [ClickHouse/ClickHouse#70420] [General]
Cannot read Iceberg table: the table schema has been changed at least 1 time, reading tables with evolved schema is supported. If you want to ignore schema evolution and read all files using latest schema saved on table creation, enable setting iceberg_engine_ignore_schema_evolution (Note: enabling this setting can lead to incorrect result). (UNSUPPORTED_METHOD) (version 24.3.5.46 (official build)) when using JDBC client in outside clickhouse server but running fine when using through terminal clickhouse-client on clickhouse server

**Comment by freeman-buzzvil** [comment]
I think this is not a JDBC error, but rather that the current ClickHouse-Iceberg engine does not support reading tables with modified schemas. In my case, I encountered the same error even when running it directly from the ClickHouse-client
